src/aggregator/client/config.go

```diff
@@ -52,10 +52,10 @@ type Configuration struct {
 	ShardCutoverWarmupDuration *time.Duration                  `yaml:"shardCutoverWarmupDuration"`
 	ShardCutoffLingerDuration  *time.Duration                  `yaml:"shardCutoffLingerDuration"`
 	Encoder                    EncoderConfiguration            `yaml:"encoder"`
-	FlushSize                  int                             `yaml:"flushSize"`
```

Comment:
Wouldn't that break parsing of existing yaml configs where this field is present? Maybe leave it with `omitempty` annotation?

Reply:
kept the old one w/ deprecated comment